api/message.go

```diff
@@ -259,11 +263,13 @@ func (c *Client) SendEmbedReply(
 // Fires a Message Create Gateway event.
 func (c *Client) SendMessage(
 	channelID discord.ChannelID, content string, embed *discord.Embed) (*discord.Message, error) {
```

Comment:
I think this can also do `SendMessage(discord.ChannelID, string, ...discord.Embed)`.